src/main/telemetry/ibus_shared.c

```diff
@@ -310,7 +310,7 @@ static uint16_t getMode()
 
 static int16_t getACC(uint8_t index)
 {
-    return (int16_t)(((float)acc.accADC[index] / acc.dev.acc_1G) * 1000);
+    return (int16_t)((acc.accADC[index] / acc.dev.acc_1G) * 1000);
```

Comment:
Does this produce a warning without explicit conversion?

Reply:
on my test FC
LUX_RACE
no.
I didn't checked the Travis logs !